### PR TITLE
Week4 js homework 

### PR DESCRIPTION
**Pull request by danzelo1**

### CODE REVIEW

File: Javascript/homework/responses/4.1-page-setup.md [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
 
+First off, copy your files from last week into your week 4 folder.  Next, add a div tag below your clock div (below the `</div>` tied to `<div class="clock">` ).  Make sure to give this tag a unique id attribute, my line of code looks like `<div class="digital" id="digital clock"></div>`.
+
+Next, copy the function you wrote for your digital clock in week 2 to show the time.  Paste the contents of this function into the function that sets the analog clock so that both clocks are set with a single function call.  Be sure to rename any repeating variables if necessary to get the function working.  Also, make sure to change the code you use to print the time to search for an element with `id="digital clock"`, so my line of code looks like `document.getElementById("digital clock").textContent = time;`. 
+
+At this point, your page should look something like this:![Screen Shot 2021-01-17 at 9.11.28 PM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-17 at 9.11.28 PM.png)
+
+Now, add a block in your css file that selects the `"digital"` class like so:
```
Comment (MinyMeep1) [inline Javascript/homework/responses/4.1-page-setup.md]: What's a block. And will this be on the sheet sheet.

File: Javascript/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Bitcamp JavaScript :rocket: 
+
```
Comment (emsesc) [inline Javascript/README.md]: We actually recently updated the README.md format. It's pretty similar, but you will just have to move some stuff around: https://github.com/bitprj/BitCamp/tree/master/repo_logistics/sample-camp

File: Javascript/homework/responses/4.1-page-setup.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+### Week 4 - Toggling between digital and analog clocks and incoporating advanced features
```
Comment (emsesc) [inline Javascript/homework/responses/4.1-page-setup.md]: Make sure all the response files are formatted like described under "The Github Action" https://github.com/bitprj/BitCamp/tree/master/repo_logistics

File: Javascript/homework/responses/4.2-more-buttons.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Next, we're going to create a set of radio buttons that allow a user to choose if they’d like to see the time on an analog clock or on a digital clock.
+
+First, open another form tag directly below the one you have for the timezone selector.  Similar to how you set up the timezone buttons, open a div tag with a unique class (I named mine "view") and insert 2 radio buttons. Make sure these radio buttons share the same `name` attribute and have unique `value` and `id` attributes.
+
+These two radio buttons should have the user input whether they'd like to see the time on a digital clock or an analog clock.  There's no need to add a submit button as we will set up our page to change on the click of the radio button rather than on the submit button.
+
+After this step, your page should look like this:
+
+![Screen Shot 2021-01-17 at 9.31.52 PM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-17 at 9.31.52 PM.png)
```
Comment (emsesc) [inline Javascript/homework/responses/4.2-more-buttons.md]: Might want to fix the images. If you're using a local markdown editor, you'll need to upload the image to Github.

**Review by emsesc — CHANGES_REQUESTED** [review]
Make sure to change the week titles to those requested here: https://github.com/bitprj/BitCamp/tree/master/repo_logistics (Under The Github Action)

A bit more formatting (more bullet points than paragraphs) would also be great! Otherwise, the content is good.

**Review by emsesc — APPROVED** [review]

**Comment by emsesc** [comment]
@danzelo1 Resolve conflicts, but everything else looks great!

**Pull request merged**